packages/video_player_avplay/tizen/src/plus_player.cc

```diff
@@ -345,7 +345,6 @@ std::pair<int64_t, int64_t> PlusPlayer::GetDuration() {
       LOG_ERROR("[PlusPlayer] Player fail to get the duration.");
       return std::make_pair(0, 0);
     }
-    LOG_INFO("[PlusPlayer] Video duration: %lld.", duration);
```

Comment:
Is there any reason for removing it?

Reply:
Because this API will be called once one second,this will result in a lot of logs, affecting debugging.